### PR TITLE
Add iOS Meetings outcome document

### PR DESCRIPTION
**Pull request by diegopetrucci**
I think a document like this is valuable, as now we work in multiple timezones and not everybody has time to partecipate in all meetings.

### CODE REVIEW

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# iOS meeting outcomes
+
+### Lunch and learn, 19 Sept 2019
+A [trello board](https://trello.com/b/7INoC8YK/improvements-architecture-process) with architecture and process improvements has been created and issues prioritized.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]: WIP as the meeting is still ongoing, it was to give an example

File: Cookbook/README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ iOS Cookbook 👩‍🍳
 * [Writing a Proposal](./Technical-Documents/WritingAProposal.md)
 * [Release Process](./Technical-Documents/ReleaseProcess.md)
 * [How to add Labels to a Pull Request](./Technical-Documents/LabelsInPRs.md)
+* [What was decided in the iOS chapter meetings](./echnical-Documents/MeetingOutcomes.md)
```
Comment (Ben-Henshall) [inline Cookbook/README.md]:
```suggestion
* [What was decided in the iOS chapter meetings](./Technical-Documents/MeetingOutcomes.md)
```

**Comment by AnnKatF** [comment]
I don't think we should have this exposed in this repo. Let's create a document in our google drive where we can manage this.

**Comment by diegopetrucci** [comment]
> I don't think we should have this exposed in this repo. 

Why? Most of our meetings are around architecture, process, style, and so on. Nothing that could leak sensitive information. And we can be explicit as to not add anything that could, or related to information that is not public yet.

File: Cookbook/Technical-Documents/MeetingOutcomes.md (hunk 1 above)
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
A [trello board](https://trello.com/b/7INoC8YK/improvements-architecture-process) with architecture and process improvements. Issues to be prioritized by @AnnKatF.
```

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# iOS meeting outcomes
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
# iOS meeting outcomes

A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.

Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
```

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.
+
+Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
+
+### Lunch and learn, 19 Sept 2019
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
### Proposal: split octopus, 19 Sept 2019
Rejected as the amount of work to move octopus to its own framework is not worth the time. Splitting octopus will, instead, be tackled squad by squad with smaller pieces of work (e.g. Payments, GPaH).

### Lunch and learn, 19 Sept 2019
```

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# iOS meeting outcomes
+
+A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.
+
+Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
+
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion

### Banter
iOS support engineer to tackle smaller BAU tickets during support down-time.
Team activities TBD.
```

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+### Banter
+iOS support engineer to tackle smaller BAU tickets during support down-time.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
iOS support engineer to tackle smaller BAU tickets during support down-time.

```

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# iOS meeting outcomes
+
+A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.
+
+Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
+
+### Banter
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
### Banter, 19 Sept 2019
```

**Comment by AnnKatF** [comment]
Please mention the meeting organiser is the one responsible for adding these notes and previously checking if they contain sensitive information.

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# iOS meeting outcomes
+
+A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.
+
+Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]:
```suggestion
Each entry should be updated by the meeting organizer after the meeting has ended. Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
```

**Comment by diegopetrucci** [comment]
@AnnKatF added https://github.com/Babylonpartners/ios-playbook/pull/239/files#diff-0a1acb101de7bf5368515266727cbb66R5. Feel free to directly change the doc if you'd prefer different wording

File: Cookbook/Technical-Documents/MeetingOutcomes.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+A place to store all the outcomes from the iOS chapter meetings, in reversed chronological order.
+
+Each entry should be updated by the meeting organizer after the meeting has ended. Please always be mindful of not leaking sensitive information here. Double-check with your teammates to be sure, if you need to.
+
+### Banter, 19 Sept 2019
+iOS support engineer to tackle smaller BAU tickets during support down-time.
+
+Team activities TBD.
+### Proposal: split octopus, 19 Sept 2019
+Rejected as the amount of work to move octopus to its own framework is not worth the time. Splitting octopus will, instead, be tackled squad by squad with smaller pieces of work (e.g. Payments, GPaH).
```
Comment (AliSoftware) [inline Cookbook/Technical-Documents/MeetingOutcomes.md]: Need a link to the proposal PR

**Review by AliSoftware — COMMENTED** [review]
Agree we need to have a document to gather meeting outcomes. Not sure the Playbook, and going thru the PR process to update the document on each meeting, is the most appropriate place for it though. A Google Doc might be easier to use and track for that kind of content – and would avoid us having to do PRs for that, and would also allow us not to have to be careful about risking to leak private info, and would actually let us mention private links and private info in those outcome summaries without restrictions.

**Comment by diegopetrucci** [comment]
I have moved the document to the shared Google Drive.

**Pull request closed without merging**